auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -168,4 +187,71 @@ private static Metadata toHeaders(@Nullable Map<String, List<String>> metadata)
     }
     return headers;
   }
+
+  public static JwtHelper createJwtHelperOrNull(ClassLoader loader) {
+    Class<?> rawServiceAccountClass;
+    try {
+      // Specify loader so it can be overriden in tests
+      rawServiceAccountClass
+          = Class.forName("com.google.auth.oauth2.ServiceAccountCredentials", false, loader);
+    } catch (ClassNotFoundException ex) {
+      return null;
+    }
+    try {
+      return new JwtHelper(rawServiceAccountClass, loader);
+    } catch (ReflectiveOperationException ex) {
+      // Failure is a bug in this class, but we still choose to gracefully recover
+      log.log(Level.WARNING, "Failure loading to be able to convert creds to JWT", ex);
```

Comment:
Failed to create JWT Helper.

The stack trace / cause will contain the rest of the info